core/util/cursor.js

```diff
@@ -213,3 +215,23 @@ export default class Cursor {
         this._canvas.style.top = this._position.y + "px";
     }
 }
+
+Cursor.none = {
+    rgbaPixels: [],
```

Comment:
Have you checked that an empty cursor works with all browsers? I'm thinking it might be better to use `cursor: none;` in those cases.

Reply:
That's irrelevant. This "cursor" is a placeholder for initialization purposes only, it is never set. There is already a check in cursor.js that replaces an empty cursor with `cursor: none`.